src/Components/Login/Login.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import style from './Login.module.scss'
+import aboutIcon from '../../assets/navbar/aboutNavIcon.png'
+import settingsIcon from '../../assets/navbar/settingsNavIcon.png'
+import statsIcon from '../../assets/navbar/statsNavIcon.png'
+import timerIcon from '../../assets/navbar/timerNavIcon.png'
+
+const Login = () => {
+  return (
+    <section className={style.loginContainer}>
+      <h2 className={style.loginHeader}>Welcome to Som Timer</h2>
+      <p className={style.appDescription}>
+        Som Timer is a timer that cares. This application serves curated wellness content on break intervals. Be mindful of rest so you can focus best!
+        <br /><br />
+        Sign in with Google to get started.
+      </p>
+      {/* This login button comes from these docs https://developers.google.com/identity/sign-in/web/sign-in; We should explore these further for methods we'll need for Login */}
+      <div className='g-signin2' data-onsuccess='onSignIn' aria-label='Google Login Button'></div>
+      <section className={style.somTimerWalkthrough}>
+        <article className={style.walkthroughArticle}>
+          <img
+            className={style.icon}
+            src={timerIcon}
+            alt="timer icon"
+          />
+          <div className={style.walkthroughText}>
+            <h3 className={style.walkthroughHeader}>Home Page</h3>
+            <p className={style.walkthroughDescription}>On the home page, you can view your current focus or break interval in progress. Here, you'll find the countdown timer and built in media player that provides your wellness content.</p>
+          </div>
```

Comment:
Wondering if we might want to be a bit specific here in terms of explaining the flow of the timer cycle. For example, something like: 'On the homepage, you can view your current focus interval in progress. When the timer runs out, this is also where you select your break interval content type and view the wellness content in our built-in media player.'